pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml

```diff
@@ -0,0 +1,245 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: rh-push-to-external-registry
+  labels:
+    app.kubernetes.io/version: "1.0.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton pipeline to release Snapshot to Quay
```

Comment:
maybe? needs update

Reply:
updated. but let me know what I am missing or not saying?